backend/composer/api/views.py

```diff
@@ -531,6 +538,14 @@ class DestinationViewSet(viewsets.ModelViewSet):
     permission_classes = [IsOwnerOfConnectivityStatementOrReadOnly]
     filterset_class = DestinationFilter
 
+    def partial_update(self, request, *args, **kwargs):
```

Comment:
issue: I understand the idea but I don't believe partial_update is triggered for all the cases where we would need to recompute the journey (for example, in the case we delete a complete Via).

We already have django signals in place that cover the exact places we need: destination_changed, via_changed, destination_from_entities_changed, destination_anatomical_entities_changed, via_from_entities_changed, via_anatomical_entities_changed, connectivity_statement_origins_changed.

 My recommendation would be to run the 
        `cs.save(update_fields=["journey_description", "journey_entities"])`
or equivalent

Reply:
yes thank you @afonsobspinto ! I made the change.